app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageAiManager.kt

```diff
@@ -204,7 +204,7 @@ class NewMessageAiManager @Inject constructor(
 
     fun observeAiFeatureFlagUpdates() {
         newMessageViewModel.currentMailboxLive.observeNotNull(viewLifecycleOwner) { mailbox ->
-            val isAiEnabled = mailbox.featureFlags.contains(FeatureFlag.AI)
+            val isAiEnabled = mailbox.local.featureFlags.contains(FeatureFlag.AI)
```

Comment:
This live data is only ever used to get the current mailbox to now if its local values has the ai flag, would it make sens to replace it with a live data that only returns ai's enabled status? You can still keep the currentMailboxLive as a private live data and map it in a second public `isAiEnabled` live data or something I guess. Or directly query the local values from realm without querying the whole mailbox if it makes sens

Reply:
This is an EmbeddedObject, we can't query it directly.